src/main/java/mekanism/common/item/interfaces/IJetpackItem.java

```diff
@@ -113,33 +117,38 @@ private static ItemStack getJetpack(LivingEntity entity, Predicate<ItemStack> ma
     /**
      * @return If fall distance should get reset or not
      */
-    static boolean handleJetpackMotion(Player player, JetpackMode mode, BooleanSupplier ascendingSupplier) {
+    static boolean handleJetpackMotion(Player player, JetpackMode mode, double thrust, BooleanSupplier ascendingSupplier) {
         Vec3 motion = player.getDeltaMovement();
-        if (mode == JetpackMode.NORMAL) {
-            if (player.isFallFlying()) {
-                Vec3 forward = player.getLookAngle();
-                Vec3 delta = forward.multiply(forward.scale(0.15))
-                      .add(forward.scale(1.5).subtract(motion).scale(0.5));
-                player.setDeltaMovement(motion.add(delta));
-                return false;
-            } else {
-                player.setDeltaMovement(motion.x(), Math.min(motion.y() + 0.15D, 0.5D), motion.z());
-            }
+        if ((mode == JetpackMode.NORMAL || mode == JetpackMode.VECTOR) && player.isFallFlying()) {
+            Vec3 forward = player.getLookAngle();
+            Vec3 drag = forward.scale(1.5).subtract(motion).scale(0.5);
+            Vec3 delta = forward.scale(thrust).add(drag);
+            player.setDeltaMovement(motion.add(delta));
```

Comment:
I believe this can be `player.addDeltaMovement(delta)`

Reply:
I think that was already there unless I did that when I vectorized it. But I can swap it. Wasn't sure if there were other side effects.

Reply:
Yeah it was already like that, I just missed it when I converted some use cases into `addDeltaMovement` from doing a manual impl of that method.